timur/lib/client/jsx/components/query/query_select_pane.tsx

```diff
@@ -48,61 +44,47 @@ const QuerySelectPane = () => {
   if (!state.rootModel) return null;
 
   return (
-    <Card>
-      <CardHeader
-        title='Select'
-        subheader='the additional models and attributes you want'
-      />
-      <CardContent>
-        {intersectionModels.map((modelName: string, index: number) => {
-          if (!state.rootModel) return;
+    <QueryClause title='Select'>
+      {intersectionModels.map((modelName: string, index: number) => {
+        if (!state.rootModel) return;
 
-          let choiceSet = [
-            ...new Set(state.graph.allPaths(state.rootModel).flat())
-          ]
-            .filter(
-              (m) => !intersectionModels.includes(m) && m !== state.rootModel
-            )
-            .concat([modelName]);
-          return (
-            <QueryModelAttributeSelector
-              key={index}
-              label='Intersection Model'
-              modelValue={modelName}
-              modelChoiceSet={choiceSet}
-              onSelectModel={(newModelName) =>
-                updateIntersectionModels(newModelName, index)
-              }
-              onSelectAttributes={setAttributes}
-              selectedAttributes={
-                modelName && state.attributes && state.attributes[modelName]
-                  ? state.attributes[modelName]
-                  : []
-              }
-              canRemove={true}
-              removeModel={() => removeIntersectionModel(index)}
-            />
-          );
-        })}
-      </CardContent>
-      <CardActions disableSpacing>
-        {state.rootModel ? (
-          <Tooltip
-            title='Add intersection model'
-            aria-label='add intersection model'
-          >
-            <IconButton
-              aria-label='add model and attributes'
-              onClick={() =>
-                setIntersectionModels(intersectionModels.concat(['']))
-              }
-            >
-              <AddIcon />
-            </IconButton>
-          </Tooltip>
-        ) : null}
-      </CardActions>
-    </Card>
+        let choiceSet = [
+          ...new Set(state.graph.allPaths(state.rootModel).flat())
+        ]
+          .filter(
+            (m) => !intersectionModels.includes(m) && m !== state.rootModel
```

Comment:
This filters out the root model, so you can't actually select attributes from the root model in the Select pane anymore. I think you'll need to remove this filter, to get the desired behavior? For example, if I select `patient` in the From pane, I can no longer select `patient` in any Select model selector.